src/Mongolid/Cursor/Cursor.php

```diff
@@ -147,6 +148,24 @@ public function disableTimeout(bool $flag = true)
         return $this;
     }
 
+    /**
+     * This describes how the Cursor route the future read operations to the members of a replica set.
+     *
+     * @see http://php.net/manual/pt_BR/class.mongodb-driver-readpreference.php
```

Comment:
@ernjs The linked documentation is in Portuguese. It should be `https://php.net/manual/en/class.mongodb-driver-readpreference.php`